ecosystem/indexer-grpc/indexer-grpc-fullnode/src/stream_coordinator.rs

```diff
@@ -154,39 +152,104 @@ impl IndexerStreamCoordinator {
                             )),
                             chain_id: ledger_chain_id as u32,
                         };
-                        match transaction_sender.send(Result::<_, Status>::Ok(item)).await {
-                            Ok(_) => {},
-                            Err(_) => {
-                                // Client disconnects.
-                                return Err(Status::aborted(
-                                    "[Indexer Fullnode] Client disconnected",
-                                ));
-                            },
-                        }
+                        responses.push(item);
                     }
                 }
-
-                log_grpc_step_fullnode(
-                    IndexerGrpcStep::FullnodeSentBatch,
```

Comment:
this step got accidentally removed, can you add it back in?

Reply:
done.